tests/test_pool.py

```diff
@@ -1,454 +1,490 @@
-
-
-@pytest.mark.asyncio
-async def test_release_closed_connection(loop, pool_maker, dsn):
-    pool = await pool_maker(loop, dsn=dsn)
-
-    conn = await pool.acquire()
-    await conn.close()
-
-    await pool.release(conn)
-    pool.close()
-
-
-@pytest.mark.asyncio
-async def test_wait_closing_on_not_closed(loop, pool_maker, dsn):
-    pool = await pool_maker(loop, dsn=dsn)
-
-    with pytest.raises(RuntimeError):
-        await pool.wait_closed()
-    pool.close()
-
-
-@pytest.mark.asyncio
-async def test_close_with_acquired_connections(loop, pool_maker, dsn):
-    pool = await pool_maker(loop, dsn=dsn)
-
-    conn = await pool.acquire()
-    pool.close()
-
-    with pytest.raises(asyncio.TimeoutError):
-        await asyncio.wait_for(pool.wait_closed(), 0.1, loop=loop)
-    await conn.close()
-    await pool.release(conn)
-
-
-@pytest.mark.parametrize('db', pytest.db_list)
-@pytest.mark.asyncio
-async def test_pool_with_executor(loop, pool_maker, dsn, executor):
-    pool = await pool_maker(loop, executor=executor, dsn=dsn, minsize=2, maxsize=2)
-
-    conn = await pool.acquire()
-    try:
-        assert isinstance(conn, Connection)
-        assert not conn.closed
-        assert conn._executor is executor
-        cur = await conn.cursor()
-        await cur.execute('SELECT 1')
-        val = await cur.fetchone()
-        assert (1,) == tuple(val)
-    finally:
-        await pool.release(conn)
-    # we close pool here instead in finalizer because of pool should be
-    # closed before executor
-    pool.close()
-    await pool.wait_closed()
-
-
-@pytest.mark.parametrize('db', pytest.db_list)
-@pytest.mark.asyncio
-async def test_pool_context_manager(loop, pool):
-    assert not pool.closed
-    async with pool:
-        assert not pool.closed
-    assert pool.closed
-
-
-@pytest.mark.parametrize('db', pytest.db_list)
-@pytest.mark.asyncio
-async def test_pool_context_manager2(loop, pool):
-    async with pool.acquire() as conn:
-        assert not conn.closed
-        cur = await conn.cursor()
-        await cur.execute('SELECT 1')
-        val = await cur.fetchone()
-        assert (1,) == tuple(val)
-
-
-@pytest.mark.parametrize('db', pytest.db_list)
-@pytest.mark.asyncio
-async def test_all_context_managers(dsn, loop, executor):
-    kw = dict(dsn=dsn, loop=loop, executor=executor)
-    async with aioodbc.create_pool(**kw) as pool:
-        async with pool.acquire() as conn:
-            async with conn.cursor() as cur:
-                assert not pool.closed
-                assert not conn.closed
-                assert not cur.closed
-
-                await cur.execute('SELECT 1')
-                val = await cur.fetchone()
-                assert (1,) == tuple(val)
-
-    assert pool.closed
-    assert conn.closed
-    assert cur.closed
+import asyncio
+
+import pytest
+import aioodbc
+from aioodbc import Pool, Connection
+from pyodbc import Error
+
+@pytest.mark.asyncio
+async def test_create_pool(loop, pool_maker, dsn):

```

Comment:
Odd, why github shows diff here?

Reply:
ah, good point, can you please convert to unix end of line?

Reply:
I updated the file but still the same issue. Do you have any other ideas?